Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM buildpack-deps:stretch
```

Comment:
Just curious, why not use [the official rust images](https://hub.docker.com/_/rust)?

Reply:
TBF, I didn't know there was one. And now that I know, [Official](https://github.com/rust-lang/docker-rust/blob/deb612dd257e15f2bfe877fd3253b2b1b7c261df/1.43.0/stretch/Dockerfile) is almost the same as ours but a with more architecture options. I would keep ours just because I'm proud and lazy haha.What do you think?

Reply:
I think that basing ourselves from the official version has some benefits:

1. Building will be faster because we have to download an image instead of downloading the image + dependencies + building.
2. It will be easier to maintain in the long run.

Reply:
I've changed it, you are right with that.